tpl/hunger/hunger.twig

```diff
@@ -34,4 +57,6 @@
     </div>
 </div>
 
+<script async defer src="//cdnjs.cloudflare.com/ajax/libs/moment.js/2.22.2/moment.min.js"></script>
+<script async defer src="//openingh.openstreetmap.de/opening_hours.js/opening_hours+deps.min.js"></script>
```

Comment:
Please directly checkin any dependancies. We don't want to use CDNs because of GDPR.

Reply:
The second library is a shit ton of code. Why do we need 80KBs of javascript to parse some dates?

Reply:
Haven't checked if everything is needed but it can parse the [openstreetmap opening_hours](https://wiki.openstreetmap.org/wiki/Key:opening_hours) format which is very flexible.

Reply:
ok included a custom build that I got boiled down to ~88kb by removing locales an school holidays.